todo/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { updateTitle } from './components/actions';
+import TodoList from './components/TodoList';
+import Title from './components/Title';
+
+
+class App extends Component {
+
+  render() {
+
+    return (
+      <div className="App">
+        
+
+        <Title />
+        <TodoList />
+
+      </div>
+    );
+  }
+}
+
+// This function states what part of the state tree should be exposed to this 
+// component.
+const mapStateToProps = (state) => {
```

Comment:
We're not doing anything with this state, so having a MSTP function in this component is probably not useful, correct?